main/main.cpp

```diff
@@ -3203,6 +3195,16 @@ Error Main::setup2(bool p_show_boot_logo) {
 		GDExtensionManager::get_singleton()->initialize_extensions(GDExtension::INITIALIZATION_LEVEL_SCENE);
 
 		OS::get_singleton()->benchmark_end_measure("Scene", "Modules and Extensions");
+
+		/// We need to initialize the movie writer here in case
+		/// One of the user provided gdextensions subclasses MovieWriter
```

Comment:
```suggestion
		// We need to initialize the movie writer here in case
		// one of the user provided gdextensions subclasses MovieWriter.
```

[Comment style](https://docs.godotengine.org/en/stable/contributing/development/code_style_guidelines.html#comment-style-guide)